base_app/src/app/app-tab-overlay/implemented-overlays.service.ts

```diff
@@ -51,7 +51,7 @@ export class ImplementedOverlaysService {
             .map(this.initData)
             .catch(this.handleError);
         */
-        return (null);
+        return null;
```

Comment:
This function is annotated as returning a 'Observable<CSSPairList>' but here you return null. It looks like this function can't return anything other than null, in which case the function signature should be `getOverlays(): void` and you can omit the return statement.

Reply:
This class is obsolete and is being removed .